content/docs/command-reference/repro.md

```diff
@@ -262,31 +263,41 @@ The answer to universe is 42
 - The Hitchhiker's Guide to the  Galaxy
 ```
 
-Now, using the `--downstream` option results in the following output:
+And add a new stage to the pipeline:
 
 ```dvc
-$ dvc repro --downstream
+$ dvc run -n final -d count.txt -o alphabet.txt \
```

Comment:
may be I'm missing something, but why do we need an additional stage?

Reply:
Because it only had 2 stages which doesn't allow us to demo the --downstream option.

Reply:
However this command `"cat count.txt | egrep -o '[a-zA-Z]+' > alphabet.txt"` is probably not very meaningful... 🧠

Reply:
it does, right? since downstream include the target stage itself. But I see it can be not so obvious. I just have a slight concern that example got bigger and more complicated .

Reply:
Hmmm it's true... the 3rd stage here doesn't really show much at the moment since nothing gets repro! We would need to also change numbers.txt manually or something like that to show that only the 2nd and 3rd stages get reproduced.

Reply:
...which makes it even longer. You call Ivan!

Reply:
p.s. the change to both files could be a single line though, instead of the "Hitchhikers..." reference:

```console
$ echo changed | tee text.txt numbers.txt
```

Reply:
>  since downstream include the target stage itself.

Yes you are correct @shcheklein. I also feel now that we can omit adding a new stage and change the `process.py` and `text.txt` files and show that `dvc repro --downstream count` only updates the count stage and not the stages above it. Thanks!